src/components/organisms/ChatSidebar/ChatSidebar.tsx

```diff
@@ -77,34 +80,59 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
           ) : (
             <>
               <FontAwesomeIcon icon={faChevronLeft} size="sm" />
-              <FontAwesomeIcon icon={faCommentDots} size="lg" />
+              <div className="chat-sidebar__controller-wrapper">
+                <FontAwesomeIcon icon={faCommentDots} size="lg" />
+                <span className="chat-sidebar__controller-text">Chat</span>
+              </div>
             </>
           )}
         </button>
-
-        {isExpanded && (
-          <div className="chat-sidebar__tabs" role="tablist">
-            <button
-              role="tab"
-              id={venueTabId}
-              aria-label={venueChatTabTitle}
-              aria-selected={isVenueChat}
-              className={venueChatTabStyles}
-              onClick={selectVenueChat}
-            >
-              {venueChatTabTitle}
-            </button>
-            <button
-              role="tab"
-              id={privateTabId}
-              aria-label={privateChatTabTitle}
-              aria-selected={isPrivateChat}
-              className={privateChatTabStyles}
-              onClick={selectPrivateChat}
-            >
-              {privateChatTabTitle}
-            </button>
-          </div>
+        {!isExpanded ? (
```

Comment:
if you do need the ternary operator, it's usually a better idea to start not from negative value
`!condition ? 1 : 2`
but 
`condition ? 2 : 1`

Reply:
fixed 837f415